tests/users/__snapshots__/test_api.ambr

```diff
@@ -94,28 +94,10 @@
   }
 ---
 # name: test_add_permission[uvloop-invalid_permission]
-  <class 'list'> [
-    <class 'dict'> {
-      'ctx': <class 'dict'> {
-        'enum_values': <class 'list'> [
-          'cancel_job',
-          'create_ref',
-          'create_sample',
-          'modify_hmm',
-          'modify_subtraction',
-          'remove_file',
-          'remove_job',
-          'upload_file',
-        ],
-      },
-      'in': 'path',
-      'loc': <class 'list'> [
-        'permission',
-      ],
-      'msg': "value is not a valid enumeration member; permitted: 'cancel_job', 'create_ref', 'create_sample', 'modify_hmm', 'modify_subtraction', 'remove_file', 'remove_job', 'upload_file'",
-      'type': 'type_error.enum',
-    },
-  ]
+  <class 'dict'> {
+    'id': 'bad_request',
+    'message': "'INVALID'",
+  }
```

Comment:
What happened here?

Reply:
The Aiohttp-pydantic validation did not work with PermissionType, so I had to use `str` and convert it to SpacePermission, raising a HTTPBadRequest manually.